packs/alertmanager/variables.hcl

```
@@ -0,0 +1,135 @@
+variable "job_name" {
+  description = "The name to use as the job name which overrides using the pack name."
+  type        = string
+  // If "", the pack name will be used
+  default = ""
+}
+
+variable "count" {
+  description = "How many instances to start."
+  type        = string
+  default = "1"
+}
+
+variable "constraints" {
+  description = "Constraints to apply to the entire job."
+  type        = list(object({
+    attribute = string
+    operator  = string
+    value     = string
+  }))
+  default = [
+    {
+      attribute = "$${attr.kernel.name}",
+      value     = "linux",
+      operator  = "",
+    },
+  ]
+}
+
+variable "container_args" {
+  description = "Arguments passed to the alertmanager docker container"
+  type        = list(string)
+  default = [
+    "--config.file=/etc/alertmanager/config/alertmanager.yml",
+  ]
+}
+
+variable "datacenters" {
+  description = "A list of datacenters in the region which are eligible for task placement."
+  type        = list(string)
+  default     = ["dc1"]
+}
+
+variable "region" {
+  description = "The region where the job should be placed."
+  type        = string
+  default     = "global"
+}
+
+variable "version_tag" {
+  description = "The docker image version. For options, see https://hub.docker.com/r/prom/alertmanager"
+  type        = string
+  default     = "latest"
```

Let's default to a non-latest tag.
```suggestion
  default     = "v0.23.0"
```